### ISSUE TITLE
Chrome 93 FID values reported are very different from CRUX report

### ISSUE THREAD

**Issue by ZeonTrevor** [GoogleChrome/web-vitals#184]
Since the release of Chrome version 93, we are seeing a dramatic increase in the FID values from the field data that we measure from the `web-vitals` library in comparison to what we see in the CRUX report for the SPA that we have.

FID values 75th percentile, the values for both phone and tablet have increased many folds:
![Screenshot 2021-10-06 at 09 04 27](https://user-images.githubusercontent.com/4203328/136156345-8a2f6b67-5a77-4025-a418-8b53caeae0a1.png)

But looking at the CRUX report results for FID values for phone in the same time period we don't see much variation or increase in the FID:
![Screenshot 2021-10-06 at 09 15 25](https://user-images.githubusercontent.com/4203328/136157690-8995092c-6d37-4305-810a-2a155e284e65.png)

Looking at the changes that went live between chrome 92 and 93, we found there was a change on how FID is computed for the desktop: https://chromium.googlesource.com/chromium/src/+/9242ae1c1080f6694ff3cbd945b38da102c36d63%5E%21/#F3

Since the FID values measured in the CRUX report has been stable over the past month since the release, we were wondering if how FID is measured in the `web-vitals` library might have to be changed to match how it is measured by Chrome ?

**Comment by philipwalton** [comment]
Hey @ZeonTrevor there were no changes released in Chrome 93 that were expected to affect FID (see the [changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/fid.md)), so this is surprising.

Do you have a public URL you could share so we could take a look to see if there's a regression?

**Comment by ZeonTrevor** [comment]
Hi @philipwalton, sorry for the delay in responding back here. Yes you are right that we were also not able to reproduce this issue while testing locally on our devices back then. 

But this topic came back in focus, as we finally saw an increase in the CRUX report data around 17th January after a delay of almost 4 months and the p75 value for FID in the end reached a max value that was similar to what we were measuring with the `web-vitals` library. 

While investigating the FID once again, we were able to reproduce the issue by scrolling fast while the page was loading. The issue we noticed was with the `touchstart` event only on mobile and tablet devices for obvious reasons.

And we were finally able to fix the FID according to the `web-vitals` library by optimising these event listeners on the page. These event-listeners were there since a long time, even when we were in the green area for FID so we are assuming that something definitely changed in the FID computation on Chrome but we don't yet have an overview in any discussion thread if this is really the case.
